_docs/concepts/policy-and-control/mixer.md

```diff
@@ -63,19 +63,13 @@ to target new or custom infrastructure backends.
 
 ## Configuration state
 
-Mixer's core runtime methods (`Check`, `Report`, and `Quota`) all accept a set of attributes on input and
-produce a set of attributes on output. The work that the individual methods perform is dictated by the set of input
-attributes, as well as by Mixer's current configuration. To that end, the service operator is responsible
-for:
+Mixer's core runtime methods ([Check](https://github.com/istio/api/blob/master/mixer/v1/service.proto#L43) and [Report](https://github.com/istio/api/blob/master/mixer/v1/service.proto#L48)) accept a set of attributes on input. Mixer's current configuration dictates the work that the individual methods perform with the set of input attributes. To that end, the service operator is responsible for:
```

Comment:
Skip the links to code. Or potentially, point to links from references section

Reply:
Removed links to code.